### ISSUE TITLE
RPC not appearing in Active Now section on discord

### ISSUE THREAD

**Issue by EduardoLozaRojas** [down-bad/advanced-rpc#6]
Hello, I was wondering if the plugin is able to display the activity on the "Active Now" section on the Friends page. 
![image](https://user-images.githubusercontent.com/73665126/208272205-5388067b-f43b-488e-a989-bebc8fbaf5c8.png)
Like you can see in the picture above, nothing is showing there. Is that normal ?
FYI : I'm ShadyLoza using Cider with AdvancedRPC on the screenshot

**Comment by down-bad** [comment]
Hey, yes it's possible for AdvancedRPC to show in the active now section, but Discord decides when and to who. This applies for all statuses, not just for this plugin.

**Comment by EduardoLozaRojas** [comment]
The thing is, when I use Cider's RPC, I can see it in the active now section, but when I turn on AdvancedRPC it just doesn't show it. That's why I was a bit confused. 

It should show it when you're friends with someone no ? 
Thank you for answering my questions btw!! Really appreciate the support

**Comment by down-bad** [comment]
The active now section is out of anyone's control I'm afraid. Discord uses various algorithms to determine what and who to show you there based on your account data and activity. Not a bug.

**Comment by EduardoLozaRojas** [comment]
Thank you for the clarification! Might be useful to keep it in mind because Cider's RPC shows it on Active Now but not AdvancedRPC.

**Comment by EduardoLozaRojas** [comment]
@down-bad I found a fix! For it to appear in Active Now, I had to reset my settings to default. 
So apparently, it only appears in the Active Now section ONLY if the Timestamp setting is turned ON. 
BUT, if I TURN OFF the Timestamp setting, the RPC disappears from the Active Now section. 
![image](https://user-images.githubusercontent.com/73665126/208316464-5754f1bd-5313-44c7-ba89-5d2e9146e587.png)

Is there a possible fix for it to appear even if Timestamp setting is turned off ? 
I suppose Discord's RPC, or at least for it to appear in Active Now, really depends on the .startTimestamp value when initialized.
